### ISSUE TITLE
Proposals fields has no limitation for size.

### ISSUE THREAD

**Issue by cryply** [gamedaoco/gamedao-beta#268]
**Describe the bug**
FE does not restrict from entering super long strings in Proposal

**To Reproduce**
1. Go to DAO you belongs
2. Click on Create Proposal
3. Fill up many Proposal Title and Proposal Content Description with long string 
4. Click on Publish Proposal
5. See that FE take long time due to fact that its actually sending this Loremipsum to BE.

**Expected behavior**
Length of Title and Description must be limited.

**Screenshots**
![image](https://user-images.githubusercontent.com/36797566/148427798-67851de8-8c82-4caa-9804-8f58e593e75b.png)

**Additional context**
I was also not authorize to Publish is it due to fact that DAO not mine? But I guess any member of DAO can make proposal...

**Comment by DarkNebula0** [comment]
@2075 Here come up 2 new questions :)

1. What is the maximum length of the text 
2. When is a member allowed to create a proposal or what kind of restriction is there?

**Comment by soundyogi** [comment]
I think 48bytes. 
```if(new Blob([values.title]).size > 48) errors.title = 'Please enter a maximum of 48 bytes.'```
I used this one in Campaigns:

(I hope I have validation done for proposals soon working again)

**Comment by cryply** [comment]
utf-8  can be up to 4 bytes per character

**Comment by 2075** [comment]
yeah, the field validations should possibly come from a lib providing this, internationalisation?

**Comment by 2075** [comment]
@DarkNebula0 
- max length should be a sane number
- a member of a dao can always create proposals
- a member of a campaign can also vote on withdrawal votings
- a controller of a dao can create withdrawal votings

all up for discussion
